pkg/client/client.go

```diff
@@ -24,6 +25,8 @@ const (
 	CONFIG_PATH     = CONFIG_DIR + "/config.yaml"
 	AUTH_PATH       = CONFIG_DIR + "/auth.json"
 	INSTANCEID_PATH = CONFIG_DIR + "/instanceid"
+	MAX_SIZE        = 5242880
```

Comment:
I would probably make the name here a bit more descriptive...

The first question I would ask when I see this is "What is this the max size for?" so probably a good idea to include a prefix appropriate to the object that it is the max size for... E.g. ITEM_MAX_SIZE etc